src/main/java/core/basesyntax/classes/figures/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.classes.figures;
+
+public class Rectangle extends Figure {
+    private final int sideA;
+    private final int sideB;
+
+    public Rectangle(String color, int sideA, int sideB) {
+        super(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    @Override
+    public double calculateArea() {
+        return this.sideA * this.sideB;
+    }
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.